core/src/test/java/org/hibernate/ogm/backendtck/storedprocedures/NamedParametersStoredProcedureCallTest.java

```diff
@@ -62,7 +62,7 @@
 	protected EntityManager em;
 
```

Comment:
Can you add comment on top of the class, arguing that even if the *INFINISPAN_REMOTE* dialect is skipped in this test, the test is provided and run by the dialect module test, using both java and javascript procedure?

Reply:
sure, I will add that

Reply:
thank you